libs/ui-components/src/components/Device/DeviceList/DeviceTableToolbar.tsx

```diff
@@ -135,10 +135,14 @@ const DeviceToolbarChips = ({
           </SplitItem>
         );
       })}
-      {fleetId && (
+      {ownerFleets.length && (
```

Comment:
To avoid a "0" appearing here as well.

```suggestion
      {!!ownerFleets.length && (
```

Reply:
fixed